### ISSUE TITLE
identifier "__INT_MAX__" is undefined

### ISSUE THREAD

**Issue by zfields** [microsoft/vscode-cpptools#1706]
macOS 10.13.3
VSCode v1.21.1
Apple LLVM version 9.0.0 (clang-900.0.39.2)

<img width="318" alt="screen shot 2018-03-20 at 1 52 50 pm" src="https://user-images.githubusercontent.com/2801887/37682077-bebe19fe-2c45-11e8-9084-125a980349fa.png">

```
{
    "configurations": [
        {
            "name": "Mac",
            "includePath": [
                "/usr/local/include",
                "/Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/include/c++/v1",
                "/Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/lib/clang/9.0.0/include",
                "/Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/include",
                "/Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX10.13.sdk/usr/include",
                "${workspaceRoot}"
            ],
            "defines": [],
            "intelliSenseMode": "clang-x64",
            "browse": {
                "path": [
                    "/usr/local/include",
                    "/Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/include/c++/v1",
                    "/Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/lib/clang/9.0.0/include",
                    "/Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/include",
                    "/Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX10.13.sdk/usr/include",
                    "${workspaceRoot}"
                ],
                "limitSymbolsToIncludedHeaders": true,
                "databaseFilename": ""
            },
            "macFrameworkPath": [
                "/Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX10.13.sdk/System/Library/Frameworks"
            ]
        },
        {
            "name": "Linux",
            "includePath": [
                "/usr/include",
                "/usr/local/include",
                "${workspaceRoot}"
            ],
            "defines": [],
            "intelliSenseMode": "clang-x64",
            "browse": {
                "path": [
                    "/usr/include",
                    "/usr/local/include",
                    "${workspaceRoot}"
                ],
                "limitSymbolsToIncludedHeaders": true,
                "databaseFilename": ""
            }
        },
        {
            "name": "Win32",
            "includePath": [
                "C:/Program Files (x86)/Microsoft Visual Studio 14.0/VC/include",
                "${workspaceRoot}"
            ],
            "defines": [
                "_DEBUG",
                "UNICODE",
                "_UNICODE"
            ],
            "intelliSenseMode": "msvc-x64",
            "browse": {
                "path": [
                    "C:/Program Files (x86)/Microsoft Visual Studio 14.0/VC/include/*",
                    "${workspaceRoot}"
                ],
                "limitSymbolsToIncludedHeaders": true,
                "databaseFilename": ""
            }
        }
    ],
    "version": 3
}
```

**Comment by sean-mcmanus** [comment]
Can you try the 0.16.0-insiders build? https://github.com/Microsoft/vscode-cpptools/releases/tag/v0.16.0-insiders ? We should guess the clang compiler, but if not you can set it via the compilerPath.

**Comment by zfields** [comment]
How? I could not find the option for switching to "insiders" in the IDE.

Why was the [previous issue](#1674) closed with a `more info needed` label, instead of being left open with `resolved` and `pending release` labels?

**Comment by sean-mcmanus** [comment]
The previous issue was closed because 0.16.0-insiders fixed the issue...the user closed it. We have other issues tracking the fix.

The insiders instructions are to "Download the .vsix that matches your OS and run the "Install from VSIX" command in VS Code.". We're planning to release the official version tomorrow (unless there's a delay).

**Comment by zfields** [comment]
I can also confirm it works, but please leave issues open and tag them appropriately until they are actually released into the wild.

> _"We have other issues tracking the fix."_

Why didn't you link them? Are you tracking it internally? If you are, then please remember those tags are for us - the community. Issues are archived and searchable. I found the [closed issue](#1674) before I posted, but I dismissed it, because I assumed the poster was mistaken or the fix had been released since it was marked closed and the tag was "more info needed".

**To summarize:** If you had done better public book keeping, you wouldn't have even had to hear 🙉this crap 💩or deal with me 😈today.

**Comment by sean-mcmanus** [comment]
We didn't close the issue, the user did. We don't usually clean up tags on closed issues. The issue we're using to track it is https://github.com/Microsoft/vscode-cpptools/issues/1293 . The fix was posted in the issue itself, so adding a link to the duplicate issue didn't seem necessary.

**Comment by zfields** [comment]
Well played @sean-mcmanus, well played...
